editor/editor_export.cpp

```diff
@@ -1351,18 +1351,21 @@ bool EditorExportPlatformPC::can_export(const Ref<EditorExportPreset> &p_preset,
 	return valid;
 }
 
-String EditorExportPlatformPC::get_binary_extension(const Ref<EditorExportPreset> &p_preset) const {
+List<String> EditorExportPlatformPC::get_binary_extension(const Ref<EditorExportPreset> &p_preset) const {
+	List<String> list;
 	for (Map<String, String>::Element *E = extensions.front(); E; E = E->next()) {
 		if (p_preset->get(E->key())) {
-			return extensions[E->key()];
+			list.push_back(extensions[E->key()]);
+			return list;
```

Comment:
Should it return as soon as it finds one, or finish populating `list` with valid extensions and only return at the end?

Reply:
I returned the list immediately to keep the logic of the function as it used to be. But I guess it should finish populating the list, I'll try it out.

Reply:
This only affects the Linux export, if I let it finish populating the list:
![screenshot from 2018-10-31 16-00-17](https://user-images.githubusercontent.com/10578225/47812172-7074d200-dd26-11e8-9b17-2fc09b3d28ae.png)

Sounds okay?

Reply:
Seems fine, though normally `x86_64` is for 64-bit and `x86` is for 32-bit -- but does it mean that prior to this change, 32-bit export would also propose only `x86`?
And now, what's the default extension when doing a 32-bit export for Linux?

Reply:
So I've just checked, if I keep the functionality as it was, returning the list immediately, the linux export gets .x86 when the export is 32-bit and .x86_64 when the export is 64-bit. Seems better this way, I'll keep functionality as it is then.